GOVERNANCE.rst

```diff
@@ -0,0 +1,183 @@
+Governance
+==========
+
+Overview
+--------
+sktime is a meritocratic, consensus-based community project. Anyone with an interest in the project can join the community, contribute to the project, and participate in the governance process. This document describes how that participation takes place and how to set about earning merit within the project community. We are particularly motivated to support new and/or anxious collaborators, people who are looking to learn and develop their skills, and anyone who has experienced discrimination in the past.
+
+In particular, we will outline
+
+* how we expect all members of the sktime community to behave,
+* what priviledges and responsibilities authors and maintainers have,
+* how we curate and select new contributions and existing functionality,
+* how we make decisions,
+* how we acknowledge contributions.
+
+
+Code of Conduct
+---------------
+We value the participation of every member of our community and want to ensure an that every contributor has an enjoyable and fulfilling experience. Accordingly, everyone who participates in the sktime project is expected to show respect and courtesy to other community members at all times.
+
+To make clear what is expected, we ask all members of the community to
+conform to our `Code of Conduct <https://github
+.com/alan-turing-institute/sktime/blob/master/CODE_OF_CONDUCT.rst>`_.
+
+Authorship and maintainership of algorithms
+-------------------------------------------
+
+Authors and maintainers of algorithms have special rights and duties with regard to changes and maintenance of their algorithms and associated documentation, including doc strings and tutorial notebooks.
+
+Algorithms are encapsulated in classes. To faciliate authorship and maintainership questions, each algorithm is written in a separate file.
+
```

Comment:
hm, I can see cases where you would want to put a "bundle" of algorithms in a file for technical reasons. E.g., if they use a common optimization core.
You could define author/maintainership at a file or folder level, by "tagging" the file or folder.

Reply:
Yes, I agree, separate folders are fine too, some algorithms may of course also re-use code from other algorithms. So this should be phrased as a guideline on how to organise algorithms, but we may have to deviate from it in some situations.

Reply:
makes sense to me - do you want to change this, since it sounds pretty strict?

Reply:
ok, new change addresses this.